modules/bmc/ssh.rb

```diff
@@ -60,6 +60,36 @@ def powerstatus
       def powercycle
         ssh(@powercycle)
       end
+
+      def ip
+        host
+      end
+
+      # the following are dummy implementations
+
+      def mac
+        ''
```

Comment:
> Testing these stubs makes no sense, this is a provider aimed for testing purposes, it has no real use.

Is this PR required then? I'm not suggesting testing stubs, but rather that SSH provider implements methods required for foreman's correct behaviour.